cpp/basix/quadrature.cpp

```diff
@@ -4976,4 +4989,11 @@ template std::vector<double> quadrature::get_gl_points(int);
 
 template std::vector<float> quadrature::get_gll_points(int);
 template std::vector<double> quadrature::get_gll_points(int);
+
+/// @cond DOXYGEN_SHOULD_SKIP_THIS
```

Comment:
I think this should go around *all* the explicit instantiations, e.g. if you look at `get_gll_points` in the generated docs, you will only see `double` at the moment, not `T`, because that was the last thing doxygen saw.

Reply:
I guess there may be other explicit instantiations in other files... worth checking (maybe not in this PR).

Reply:
I've opened #929 to remind me to do this